src/lib/BUILD.gn

```diff
@@ -53,10 +54,33 @@ static_library("lib") {
   }
 
   cflags = [ "-Wconversion" ]
+}
+
+static_library("lib") {
+  public_deps = [ ":base" ]
+
+  if (chip_device_platform == "darwin") {
```

Comment:
Why darwin is forces to use stdio here? There is a dedicated `lib_using_stdio` now.

Reply:
I don't remember, why I put that there. After investigating, it seems not to be needed, so I removed it.